<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpTo</class>
 <widget class="QDialog" name="JumpTo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Location</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,2,1,2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_X">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_Y">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_Z">
       <property name="text">
        <string>Z</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Block">
       <property name="text">
        <string>Block</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_Block_X">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-30000128</number>
       </property>
       <property name="maximum">
        <number>30000128</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_Block_Y">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBox_Block_Z">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-30000128</number>
       </property>
       <property name="maximum">
        <number>30000128</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Chunk">
       <property name="text">
        <string>Chunk</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_Chunk_X">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-1875008</number>
       </property>
       <property name="maximum">
        <number>1875008</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_Chunk_Y">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinBox_Chunk_Z">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-1875008</number>
       </property>
       <property name="maximum">
        <number>1875008</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Region">
       <property name="text">
        <string>Region</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_Region_X">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-58594</number>
       </property>
       <property name="maximum">
        <number>58594</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkBox_Use_Y">
       <property name="toolTip">
        <string>Respect Y values</string>
       </property>
       <property name="text">
        <string>Use Y</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinBox_Region_Z">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-58594</number>
       </property>
       <property name="maximum">
        <number>58594</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Get">
       <property name="text">
        <string>Get location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Sync">
       <property name="toolTip">
        <string>Update values through map movement</string>
       </property>
       <property name="text">
        <string>Sync map</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Jump">
       <property name="text">
        <string>Jump to location</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinBox_Block_X</tabstop>
  <tabstop>spinBox_Block_Y</tabstop>
  <tabstop>spinBox_Block_Z</tabstop>
  <tabstop>spinBox_Chunk_X</tabstop>
  <tabstop>spinBox_Chunk_Y</tabstop>
  <tabstop>spinBox_Chunk_Z</tabstop>
  <tabstop>spinBox_Region_X</tabstop>
  <tabstop>checkBox_Use_Y</tabstop>
  <tabstop>spinBox_Region_Z</tabstop>
  <tabstop>pushButton_Get</tabstop>
  <tabstop>checkBox_Sync</tabstop>
  <tabstop>pushButton_Jump</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_Use_Y</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_Y</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Use_Y</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_Block_Y</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Use_Y</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_Chunk_Y</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_Sync</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_Get</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
